src/src/components/navigation/NavigationBar.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="navigation-bar ">
+    <div class="navigation-content">
+      <div class="navigation-logo">
+        <fa icon="fingerprint" class="navigation-icon"></fa>
```

Comment:
Codacy has a fix for the issue: [Require self-closing on Vue.js custom components (<fa>).](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
        <fa icon="fingerprint" class="navigation-icon"/>
```